pkgs/games/frotz/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchgit, ncurses }:
+
+stdenv.mkDerivation rec {
+  version = "2.44";
+  name = "frotz-${version}";
+
+  src = fetchgit {
```

Comment:
It has releases: https://github.com/DavidGriffith/frotz/releases
But even if does not have one, it would still work because `rev` can point to tags too.